docs/reference/advanced_configuration.rst

```diff
@@ -174,10 +171,6 @@ To create your own RouteBuilder create the PHP class and register it as a servic
             <argument type="service" id="sonata.admin.audit.manager" />
         </service>
 
-* YAML service registration
-
-.. configuration-block::
```

Comment:
Why remove this? Doesn't it introduce an indention issue?

Reply:
No, as it is grouped by the `.. configuration-block::` above